README.md

```diff
@@ -16,6 +16,12 @@ easy.  Just do the following:
 7. Run `./manage.py runserver`
 8. Open a browser and point it at http://localhost:8000/
 
+## Docker Version
+
+1. docker build . -t ripe-atlas-halo
```

Comment:
This didn't work for me as docker complained that `build` required an argument.  This however did work (note the placement of `.`:

```bash
docker build -t ripe-atlas-halo .
```